crates/rattler_repodata_gateway/src/lib.rs

```diff
@@ -25,7 +25,7 @@
 //! Below is a basic example that shows how to retrieve and cache the repodata for a conda channel
 //! using the [`fetch::fetch_repo_data`] function:
 //!
-//! ```rust
+//! ```no_run
```

Comment:
The example will fail if run because the `current_repo_data` file does not exist. Not really sure how to pull that off without including a ton of setup code. There are options to hide certain lines if I'm not mistaken. So maybe that would work.

Reply:
Im not sure I understand why this would fail? Its invoking `fetch::fetch_repo_data` with default arguments. Why does that fail all of a sudden?

Reply:
I thought this was in reference to the example in the `jlap/mod.rs` file (wasn't looking at the filename 😂).

The reason I changed that example is that the code sample makes an external request for a file that's pretty large. This slows down testing time, so I figured it would be good to just compile and not actually run it.

Do you still think it's a good idea to revert?